Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -185,22 +186,17 @@ export default class CovidCardView extends Vue {
             assessmentDetailsPromise,
         ])
             .then(([searchResult]) => {
+                this.searchResult = searchResult;
+                this.setAddress(
+                    this.searchResult?.patient?.postalAddress,
+                    this.searchResult?.patient?.physicalAddress
+                );
                 if (searchResult.blocked) {
-                    this.searchResult = null;
-                    this.showBannerFeedback({
-                        type: ResultType.Error,
-                        title: "Search Error",
-                        message:
-                            "Unable to retrieve record for this individual",
-                    });
+                    this.isSearchIsBlocked = true;
                 } else {
+                    this.isSearchIsBlocked = false;
                     this.phn = "";
-                    this.searchResult = searchResult;
                     this.maskHdid = true;
```

Comment:
Since we're now treating blocked responses as valid results, we should be updating the `phn` and `maskHdid` fields for them as well.